src/py21cmfast/src/HaloBox.c

```diff
@@ -288,6 +288,20 @@ int set_fixed_grids(double M_min, double M_max, InitialConditions *ini_boxes, fl
     set_integral_constants(&integral_cond, consts->redshift, M_min, M_max, M_cell);
     double growthf = dicke(consts->redshift);
 
+    // If our scaling relations define a median, the scatter will will increase the mean value
+    // due to the asymmetry of the lognormal distribution, we mimic this in the
+    // sub-sampler component. NOTE: this will also occur if FIXED_HALO_GRIDS is true, and it's
+    // not obvious whether that's what a user would want.
+    // TODO: redesign the USE_HALO_FIELD, FIXED_HALO_GRIDS, SCALING_RELATIONS_MEDIAN logic
```

Comment:
This seems kind of critical -- if this is something we want to do, then we probably should do it before v4 or else we'll have major API problems...